src/lib/email.ts

```diff
@@ -0,0 +1,33 @@
+import { env } from 'process'
+
+import fetch from 'node-fetch'
+
+interface EmailRequestOptions<T> {
+  from: string
+  to: string
+  subject: string
+  body: T
+}
+
+export const sendEmail = async <T>(template: string, { from, to, subject, body }: EmailRequestOptions<T>) => {
```

Comment:
A jsdoc comment here would be great, maybe with some usage examples + a link to docs.netlify.com

Reply:
Fantastic suggestion